src/static_deps.mli

```diff
@@ -2,6 +2,10 @@ open! Import
 
 (** A simple wrapper around [Deps.t], where some dependencies are recorded as
     "rule deps" and other as "action deps". *)
+
+(* Action Dependencies: Files the external commands are expected to access *)
+(* Rule Dependencies: Files needed in order to compute the action to execute and its dependencies *)
+
```

Comment:
Could this be added to the above docstring?

Reply:
Sure, done